examples/visualization/styles/helpers/color-continuous-style.html

```diff
@@ -45,7 +45,10 @@
         }
       });
 
-      const style = carto.viz.colorContinuousStyle('value');
+      const style = carto.viz.colorContinuousStyle('value',{
+        rangeMin: 80,
```

Comment:
I guess we need to specify them cause helper is still on manual mode, right. Are we already able to get those min / max from the source?

Reply:
Yes, if you don't specify it we get the defaults: https://github.com/CartoDB/toolkit/blob/alasarr/styles-refactor/packages/viz/src/lib/style/helpers/color-continuous-style.ts#L11